torch/testing/_internal/common_utils.py

```diff
@@ -1804,6 +1805,18 @@ def get_int64_dtype(dtype):
                                             dtype=int64_dtype, layout=layout, device=device, requires_grad=False),
                             int64_dtype, layout, device, fv + 5, False)
 
+# this helper method is to clone the input of test function
+# for the test file such as `test_ops.py` and `test_op_aliases.py`
+def clone_input_helper(input):
+    if isinstance(input, Sequence):
+        # use torch.clone for tensor, copy.deepcopy otherwise
+        return list(map(lambda x: torch.clone(x) if isinstance(x, torch.Tensor) else deepcopy(x),
```

Comment:
Just map clone_input_helper so you're calling this recursively on sequences (that would also handle nested sequences)

Reply:
Sorry for not considering the function of this helper carefully, the helper has been updated as a recursive method to support `nested sequences`.